includes/classes/class-omise-image.php

```diff
@@ -8,11 +8,13 @@ class Omise_Image {
 		 *
 		 * @param string $file Image file name with extension such as image.jpg
 		 * @param string $alternate_text Alternate text for the image
+		 * @param int $width Width size for image with default value at 30
+		 * @param int $height Max-height size for image with default value at 30
 		 * @return string HTML &lt;img&gt; element
 		 */
-		public static function get_image( $file, $alternate_text ) {
-			$url = WC_HTTPS::force_https_url( plugin_dir_url( '' ) . 'omise/assets/images/' );
-			return "<img src='$url/$file' class='Omise-Image' style='width: 30px; max-height: 30px;' alt='$alternate_text' />";
+		public static function get_image( $file, $alternate_text, $width = 30, $height =30 ) {
```

Comment:
```suggestion
		public static function get_image( $file, $alternate_text, $width = 30, $height = 30 ) {
```

should there be space here too?

Reply:
yeah you right nice catch!! 👍